Zadaha26.py

```diff
@@ -0,0 +1,9 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа A и B, и возводит число А в целую степень B с помощью рекурсии.
+x = int(input(' Введите число A: '))
+y = int(input(' Введите число B: '))
+def pov(a, n):
+ if (n == 1):
+  return a
+ else:
+  return a * pow(x, n - 1)
+print(pov(x, y))

```

Comment:
Во-первых, у вас тут неверные отступы. Во-вторых, скобки в условиях не нужны. В третьих конструкция `else` также не нужна. В четвертых, не забывайте приводить в PEP8.
```suggestion
x = int(input(' Введите число A: '))
y = int(input(' Введите число B: '))


def pov(a, n):
    if n == 1:
        return a

    return a * pow(x, n - 1)


print(pov(x, y))
```
Ну и в пятых, если передать вторым аргументам ноль, то вы получите переполнение стека рекурсии. При запуске.